CMakeLists.txt

```diff
@@ -151,6 +151,7 @@ option(FLB_IN_EMITTER         "Enable emitter input plugin"         Yes)
 option(FLB_IN_NODE_EXPORTER_METRICS "Enable node exporter metrics input plugin" Yes)
 option(FLB_IN_WINDOWS_EXPORTER_METRICS "Enable windows exporter metrics input plugin" Yes)
 option(FLB_IN_OPENTELEMETRY            "Enable OpenTelemetry input plugin"         Yes)
+option(FLB_IN_LOGCAT          "Enable Logcat input plugin"          Yes)
```

Comment:
Need to make sure we consider this appropriate particularly for the non Linux targets. Should it be for all by default? 

Do we need to customize the MacOS, Windows and other configurations in the `cmake` directory?